boot/bootutil/src/swap_scratch.c

```diff
@@ -73,6 +73,12 @@ boot_read_image_header(struct boot_loader_state *state, int slot,
         rc = BOOT_EFLASH;
     }
 
+    if (out_hdr->ih_magic != IMAGE_MAGIC) {
+       /*  image is not valid, unset all flags to avoid futher
```

Comment:
```suggestion
       /*  Image is not valid, unset all flags to avoid further
```
nit: Typo in `further` and capital initial on `Image`.